paper/main.tex

```diff
@@ -1235,15 +1219,9 @@ \subsubsection*{Synthetic data: simulating dynamic high-order
 timeseries, collectively reflecting a total of 4 qualitatively
 different patterns of dynamic correlations (i.e., 100 datasets
 reflecting each type of dynamic pattern: constant, random, ramping,
-and event).  Each timeseries comprised 10 zero-order features (dimensions)
-that varied over 300 timepoints.  After applying our dynamic
-correlation estimation procedure, this yielded a 100-dimensional
-timeseries of first-order
-features that could then be used to estimate dynamic second-order
-correlations.  (We chose to use $K=10$ zero-order features for our
-higher order simulations in order to put the accuracy computations
-displayed in Figs.~\ref{fig:synthetic} and~\ref{fig:higher_order_sims}
-on a roughly even footing.)
+and event).  Each timeseries comprised 10 features (dimensions)
+that varied over 300 timepoints for first-order correlations, and
+therefore 100 second-order features. 
 
```

Comment:
revert back to previous version

Reply:
I think I reverted back to what you want, but you might want to check me on that.